src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,47 @@
+package duke.task;
+
+/**
+ * Task object class.
+ * Consists of the task's name and if it is done.
+ */
+public class Task {
+    private final String name;
+    private boolean isDone;
+
+    public Task(String taskName) {
+        this.name = taskName;
+        this.isDone = false;
+    }
+
+    public Task(String taskName, boolean isDone) {
+        this.name = taskName;
+        this.isDone = isDone;
+    }
+
+    public String getTaskName() {
+        return this.name;
+    }
+
+    public boolean getIsDone() {
+        return this.isDone;
+    }
+
+    /**
+     * Mark the task as done
```

Comment:
Consider adding period at the end of the description. This applies to line 37 in this doc aswell.

Reply:
Noted. Thanks for pointing it out!